### PR TITLE
Convert GuessControl and NumberGussingGame class to functional component

### PR DESCRIPTION
**Pull request by JafariM**

### CODE REVIEW

File: src/GuessControl.js [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+
+  return (
+    <div>
+      <input type="number" value={currentGuess} onChange={handleInputChange} />
+      <Button onClick={onSubmitGuess}>Submit Guess</Button>
+    </div>
+  );
+};
+
+class GuessControlOld extends Component {
```
Comment (XeniyaDob) [inline src/GuessControl.js]: Great, you created a functional component. You can delete this class component.

File: src/GuessControl.js [hunk 2]
```diff
@@ -1,7 +1,28 @@
 import React, { Component } from "react";
```
Comment (XeniyaDob) [inline src/GuessControl.js:1]: You can delete everything related to the class component

**Review by XeniyaDob — CHANGES_REQUESTED** [review]
Great job! I added a couple of comments. One important thing, you need to push your changes to your main branch, not CTD main.

**Comment by JafariM** [comment]
Thank you for your review, I will fixed the issues

**Pull request closed without merging**